javascript/ql/src/Security/CWE-730/examples/server-crash.BAD.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express"),
+      fs = require("fs");
+
+function save(rootDir, path, content){
+	if (!isValidPath(rootDir, req.query.filePath)) {
+		throw new Error(`Invalid filePath: ${req.query.filePath}`); // BAD crashes the server
+	}
+	// write content to disk
+}
+express().post("/save", (req, res) => {
+	fs.access(rootDir, (err) => {
```

Comment:
(Optional). 
I think an example based on `fs.readFile()` instead of `fs.access()` might work better.   

It is immediately obvious to everyone what `fs.readFile()` does. and I don't think the same is the case for `fs.access()` (I had to look it up to be sure).